cmd/importer/stats.go

```diff
@@ -83,3 +85,51 @@ func (h *histogram) randInt() int64 {
 	}
 	return h.Bounds.GetRow(idx).GetInt64(0)
 }
+
+func getValidPrefix(lower, upper string) string {
+	for i := range lower {
+		if i >= len(upper) {
+			log.Fatalf("lower %s is larger than upper %s", lower, upper)
+		}
+		if lower[i] != upper[i] {
+			randCh := uint8(rand.Intn(int(upper[i]-lower[i]))) + lower[i]
+			newBytes := make([]byte, i, i+1)
+			copy(newBytes, lower[:i])
+			newBytes = append(newBytes, byte(randCh))
+			return string(newBytes)
+		}
+	}
+	return lower
+}
+
+func (h *histogram) getAvgLen(maxLen int) int {
+	l := h.Bounds.NumRows()
+	totalLen := 0
+	for i := 0; i < l; i++ {
+		totalLen += len(h.Bounds.GetRow(i).GetString(0))
+	}
+	avg := totalLen / l
```

Comment:
Is it possible that `l` is 0?

Reply:
no, If length of bucket is equal to 0, we will not set the histogram.